awsms-poc-alb.yml

```diff
@@ -30,9 +33,11 @@ Resources:
       SecurityGroupIngress:          
             - CidrIp: 0.0.0.0/0
               IpProtocol: -1

```

Comment:
Any specific reason we are allowing all ports access to load balancer? Can this be locked down to ports that we need at ALB ?